src/broadcast/types.ts

```diff
@@ -22,13 +22,15 @@ export enum BroadcastEvent {
   DOLPHIN_STATUS_CHANGE = "DOLPHIN_STATUS_CHANGE",
   ERROR = "ERROR",
   LOG = "LOG",
+  RECONNECT_1006 = "RECONNECT_1006",
```

Comment:
I'm not sure what the 1006 number is referring to, probably the specific error code that it returns, but as a name, it's not very useful. Like seeing `RECONNECT_1006` isn't very descriptive. Can we rename this to something that is more clearer to what it represents. Such that when someone sees that there is a susbcription to the `RECONNECT_1006` event, they would know have a better understanding about when/what it means when it gets triggered.

This goes for all references to 1006 in this PR. Like unless 1006 is some universal number meaning some specific instance of reconnection that I'm out of the loop on, I think it would be best if references to this 1006 number was something more descriptive.